packages/swapper/src/swappers/ChainflipSwapper/utils/getQuoteOrRate.ts

```diff
@@ -297,6 +297,13 @@ export const getQuoteOrRate = async (
         singleQuoteResponse.boostQuote.egressAmountNative!,
       )
 
+      const buyAmountBeforeFeesCryptoBaseUnit = toBaseUnit(
+        bnOrZero(singleQuoteResponse.boostQuote.ingressAmount!).times(
```

Comment:
suggestion: use `ingressAmountNative` to avoid `toBaseUnit()` 
<img width="627" alt="image" src="https://github.com/user-attachments/assets/edbbce2e-a4e4-4c86-bae7-2decb66fd665" />

Reply:
https://github.com/shapeshift/web/pull/8556#discussion_r1914644210